saleor/graphql/views.py

```diff
@@ -26,6 +29,27 @@
 handled_errors_logger = logging.getLogger("saleor.graphql.errors.handled")
 
 
+def tracing_wrapper(execute, sql, params, many, context):
+    if not settings.ENABLE_OPENTRACING:
+        return execute(sql, params, many, context)
+
+    start = time.time()
+    with opentracing.tracer.start_span(operation_name="query") as span:
+        span.set_tag("sql", sql)
+        span.set_tag("many", many)
+        try:
+            result = execute(sql, params, many, context)
+        except Exception as e:
+            span.set_tag("status", "error")
+            span.set_tag("exception", e)
+            raise
+        else:
+            span.set_tag("status", "ok")
+            return result
+        finally:
+            span.set_tag("duration", time.time() - start)
```

Comment:
We don't need to track the duration manually, all spans record timestamps of their beginning and end.